frontend/packages/dev-console/integration-tests/tests/pipeline.scenario.ts

```diff
@@ -120,4 +133,70 @@ describe('Pipeline', async () => {
       true,
     );
   });
+  it('start pipeline scenario', async () => {
+    let runName: string;
+    await browser.get(`${appHost}/k8s/cluster/projects/${testName}`);
+    await switchPerspective(Perspective.Developer);
+    await browser.wait(until.visibilityOf(pageSidebar));
+    await pipelinecheckStatus();
+    await execSync(
+      `oc create -f ./packages/dev-console/integration-tests/views/sample-app-pipeline.yaml -n ${testName}`,
+    );
+    await browser.wait(until.visibilityOf(pipelineTable));
+    expect(
+      pipelineTableBody.element(by.css('[data-test-id="sample-app-pipeline"]')).isPresent(),
```

Comment:
The definition of the UI element being accessed should be in the test's corresponding view file and not here in the test code.